me.eastsun.hackdesk.yml

```diff
@@ -0,0 +1,24 @@
+app-id: me.eastsun.hackdesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: hackdesk
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland # Permission needed to show the window
+  - --socket=x11 # Permission needed to show the window
+  - --socket=fallback-x11
+  - --filesystem=home/.hackdesk:create # Permission needed to save the config
+
+modules:
+  - name: hackdesk
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 hackdesk.AppImage /app/bin/hackdesk
```

Comment:
this won't work at all.

Reply:
Sorry about that. I wasn't familiar with Linux. I thought AppImages could be directly installed.
I'll do some research and come back. Thanks for the review!

Reply:
build from source.